cmd/cluster-watchall/main.go

```diff
@@ -251,6 +256,14 @@ func (acw *allClustersWatcher) NewClusterWatcher(clusterName logicalcluster.Name
 	return cw
 }
 
+func (acw *allClustersWatcher) writeSubresources(clusterName logicalcluster.Name, gv schema.GroupVersion, subs []*urmetav1a1.APIResourceSpec, prefix string) {
+	for _, sub := range subs {
+		informable := verbsSupportInformers(sub.Verbs)
+		acw.WriteCSV([]string{"SUBRESOURCE", clusterName.String(), gv.Group, gv.Version, sub.Kind, prefix + sub.Name, fmt.Sprintf("%v", informable)})
+		acw.writeSubresources(clusterName, gv, sub.SubResources, prefix+sub.Name+"/")
```

Comment:
do we assume includeSubresources means recursive over all ?  maybe this should be an additional option and single level (at any level) should be achieved by specifying the subresource as the resource and setting the  includeSubresources to true.

Reply:
Does Kubernetes has sub resources under sub resources ? What is the use case for that ?

Reply:
@pdettori : I do not know whether it actually happens, but the API seems to say that it can.  See https://github.com/kubernetes/client-go/blob/v0.28.1/dynamic/interface.go#L34, for example.

@ezrasilvera : I am not sure I follow what you are asking and suggesting.  See my response to Paolo.  The API change here follows that possibility of more than one level of "sub".  I expect that the API here will serve #772 .